extensions-core/druid-bloom-filter/src/test/java/org/apache/druid/query/filter/sql/BloomDimFilterSqlTest.java

```diff
@@ -150,12 +153,21 @@ public void testBloomFilterExprFilter() throws Exception
                   .dataSource(CalciteTests.DATASOURCE1)
                   .intervals(querySegmentSpec(Filtration.eternity()))
                   .granularity(Granularities.ALL)
-                  .filters(
-                      new ExpressionDimFilter(
-                          StringUtils.format("(bloom_filter_test(concat(\"dim2\",'-foo'),'%s') == 1)", base64),
+                  .virtualColumns(
+                      new ExpressionVirtualColumn(
```

Comment:
This test was testing the `bloom_filter_test` expression macro function rather than a generic bloom filter sql test, so I think the query needs to change to continue to do that, or this test be removed.